lib/system.nim

```diff
@@ -3756,6 +3756,16 @@ proc instantiationInfo*(index = -1, fullPaths = false): tuple[
   ##     testException(IndexError, tester(1))
   ##     # --> Test failure at example.nim:20 with 'tester(1)'
 
+proc lineInfoToString*(file: string, line, column: int): string =
+  when defined(lineInfoToStringSquashForTesting):
+    # useful for for unittests, see tfailedassert.nim
+    "FAKEFILE"
+  else:
+    file & "(" & $line & ", " & $column & ")"
+
+proc `$`*(info: type(instantiationInfo(0))): string =
```

Comment:
Sorry, but no. Not two more system.nim entries.

Reply:
done, removed 2nd one; 1st one needed to keep things DRY (used in 3 places)